packages/client/src/schema-component/antd/page/Page.tsx

```
@@ -167,75 +169,24 @@ const PageHeader = (props) => {
+          type: 'void',
+          name: 'icon' + item.name,
+          'x-component': item.component,
+          'x-component-props': {},
+        } as ISchema;
+        const componentProps = { ...props, fieldSchema };
+        return (
+          <PageExtendComponentProvider {...componentProps}>
+            <SchemaComponent schema={schema} />
+          </PageExtendComponentProvider>
+        );
+      })}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)